classes/class-wc-connect-taxjar-integration.php

```diff
@@ -311,7 +311,7 @@ public function get_supported_countries() {
 	/**
 	 * Check if a given country is supported by TaxJar.
 	 *
-	 * @param $country Two character country code.
+	 * @param $country string Two character country code.
```

Comment:
Good change! Just a note on the order of elements:

```suggestion
	 * @param string $country Two character country code.
```